azurerm/internal/services/netapp/resource_arm_netapp_volume.go

```diff
@@ -388,3 +388,23 @@ func flattenArmNetAppVolumeExportPolicyRule(input *netapp.VolumePropertiesExport
 
 	return results
 }
+
+func flattenArmNetAppVolumeMountTargets(input interface{}) []interface{} {
+	results := make([]interface{}, 0)
+	if input == nil {
+		return results
+	}
+
+	for _, item := range input.([]interface{}) {
+		if item != nil {
+			v := item.(map[string]interface{})
+			ipAddress := v["ipAddress"]
```

Comment:
this needs to be:

```suggestion
			ipAddress := v["ip_address"].(string)
```

Reply:
It cannot be updated to "ip_address" since api returns ""mountTargets":[{"provisioningState":"Succeeded","ipAddress":"10.0.2.4"}]" for property mountTargets.

Reply:
ah so this nested object isn't modelled as an object in the Swagger? in which case can we send a PR to the Swagger repo to fix that

Reply:
When it comes to `mountTargets`, it reminds me that I changed that some time ago... and it may be a mistake. I will take a look at this and see how to change it correctly (and when changing the swagger we should let the service team know and approve, which may take a long time).